portafly/src/i18n/i18n.tsx

```diff
@@ -1,36 +1,30 @@
 import i18n, { FormatFunction } from 'i18next'
 import LanguageDetector from 'i18next-browser-languagedetector'
 import { initReactI18next } from 'react-i18next'
-import { EN } from 'i18n/locales'
+import { ITranslationsPages, Translations, EN } from 'i18n'
 
 const formatFn: FormatFunction = (value, format) => {
   if (format === 'uppercase') return value.toUpperCase()
   if (format === 'lowercase') return value.toLowerCase()
   return value
 }
 
+const sections: Array<ITranslationsPages> = ['shared', 'overview', 'analytics', 'applications', 'integration']
```

Comment:
so this will be a growing list, right? That probably needs to be repeated somewhere else, like for navigation? Maybe we can define it once somewhere and take it from there? (or maybe that doesn't make sense at all)

Reply:
There is a way to auto-define this list but it's kind a ugly and doesn't even have type checking. This list coincides with `ItranslationsPage` really and it comes from reading the keys from the translations JSON.
TLDR: `sections = Object.keys(Translations.en)` and we don't need to manage any array but... it's ugly